### PR TITLE
Added issue and pull request for the ease of creating one

### LINKED ISSUE #154: Add issue and pr template
#### I am submitting a -
* [x]  **Feature Request**
* [] **Bug Report**

#### Describe your bug/feature -
Currently there is no template for issue and pull request and so there are problems in a well formatted issue which can be easy to understand. So there must be a template for both.

#### Are you working on this issue? (Yes/No)
Yes

### PR DESCRIPTION
**Pull request by creyente1897**
**Issue Number** #154 

#### Describe the changes you have made in this pr -
* Added issue template
* Added pull request template

Fixes #154

### CODE REVIEW

**Comment by satu0king** [comment]
@tachyons does this look good?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#### I am submitting a -
+* [] **Feature Request**
+* [] **Bug Report**
+
+#### Describe your bug/feature -
+
+#### Current behaviour of the bug -
+
+#### Expected behaviour of the bug -
```
Comment (tachyons) [inline .github/ISSUE_TEMPLATE.md]: "Expected behavior of the bug" looks odd to me. Just Current behaviour and expected behavior is enough

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+#### Screenshots of the bug -
```
Comment (tachyons) [inline .github/ISSUE_TEMPLATE.md]: Screenshot if relevant

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (creyente1897) [reply .github/ISSUE_TEMPLATE.md]: @tachyons Ok will do so

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (creyente1897) [reply .github/ISSUE_TEMPLATE.md]: @tachyons Got it

**Pull request merged**